ui/src/identity.ts

```diff
@@ -30,41 +30,25 @@ export const identity = identityStore.readable;
 
 // EVENTS
 enum Kind {
-  Create = "CREATE",
   Fetch = "FETCH",
 }
 
-interface Create extends event.Event<Kind> {
-  kind: Kind.Create;
-  handle: string;
-}
-
 interface Fetch extends event.Event<Kind> {
   kind: Kind.Fetch;
   urn: string;
 }
 
-type Msg = Create | Fetch;
+type Msg = Fetch;
 
 interface CreateInput {
   handle: string;
+  passphrase: string;
 }
 
 export const fetch = event.create<Kind, Msg>(Kind.Fetch, update);
 
 function update(msg: Msg): void {
   switch (msg.kind) {
-    case Kind.Create:
-      creationStore.loading();
```

Comment:
Is the creation store no longer needed?

Reply:
I had to switch to regular `async` code due to the animations.
See: https://github.com/radicle-dev/radicle-upstream/pull/837/files#diff-9adfc54888e4c85bb6cf99eb7a4cb532R40.